book/src/audit-entries.md

```diff
@@ -52,45 +52,3 @@ This field is optional, but encouraged for two reasons:
 
 An optional free-form string containing any information the auditor may wish to
 record.
-
-## `dependency-criteria`
-
-An optional inline table specifying the criteria the vetting algorithm should
-check for in a dependency subtree.
-
-Ordinarily, when vetting a crate for criteria `foo`, `cargo vet` will
-recursively vet each direct dependency for `foo` as well. This is usually what
-you want, but occasionally you may wish to add or remove criteria for certain
-subtrees.
-
-For example, a dependency used to encrypt sensitive data might need review from
-cryptography experts:
-
-```
-[audit.mynetworkingcrate]
-version = '2.3.4'
-dependency_criteria = { hmac: ['safe-to-deploy', 'crypto-reviewed'] }
```

Comment:
It seems like the format is wrong. I tried this syntax, and the toml parser insists that i replace the ":" with a "=":
`
dependency_criteria = { hmac = ['safe-to-deploy', 'crypto-reviewed'] }
`

Reply:
Good catch, fixed.